addons/website/static/tests/interactions/snippets/form.test.js

```diff
@@ -97,6 +97,45 @@ const formTemplate = /* html */ `
     </div>
 `;
 
+function createFileUploadForm(allowMultiple = false, maxFiles = 1) {
+    const multipleAttr = allowMultiple ? "multiple" : "";
+    return `
+        <div id="wrapwrap">
+            <section class="s_website_form" data-vcss="001" data-snippet="s_website_form" data-name="Form">
+                <div class="container-fluid">
+                    <form action="/website/form/" method="post" enctype="multipart/form-data"
+                          class="o_mark_required" data-mark="*" data-pre-fill="true"
+                          data-model_name="mail.mail" data-success-mode="redirect"
+                          data-success-page="/contactus-thank-you">
+                        <div class="s_website_form_rows row s_col_no_bgcolor">
+                            <div data-name="Field" class="s_website_form_field mb-3 col-12
+                                s_website_form_custom s_website_form_required pb0 o_colored_level"
+                                data-type="binary">
+                                <div class="row s_col_no_resize s_col_no_bgcolor">
+                                    <label class="col-sm-auto s_website_form_label" style="width: 200px" for="o3xe8o85w0ct">
+                                        <span class="s_website_form_label_content">File Upload</span>
+                                        <span class="s_website_form_mark">*</span>
+                                    </label>
+                                    <div class="col-sm">
+                                        <div class="o_files_zone row gx-1"></div>
+                                        <input type="file" class="form-control s_website_form_input"
+                                            name="File Upload" required id="o3xe8o85w0ct"
+                                            ${multipleAttr} data-max-files-number="${maxFiles}" data-max-file-size="64">
```

Comment:
```suggestion
                                            ${maxFiles > 1 ? "multiple" : ""} data-max-files-number="${maxFiles}" data-max-file-size="64">
```
 and no need for `allowMultiple` nor `multipleAttr`